index.html

```diff
@@ -208,8 +208,7 @@ <h2 class="modal-project-name" id="modal-project-name">
         tenetur doloremque explicabo eum, odio molestiae ea ipsum modi officia. <br><br>
 
         Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ullam odio quas expedita dolorum voluptate quibusdam
-        laborum deleniti ipsum sapiente tempora. Provident ipsum iusto eum reiciendis pariatur praesentium nisi iste
-        doloremque.
+        laborum deleniti ipsum sapiente tempora.
       </p>
       <div class="modal-button-container">
         <button class="modal-link-button-1 modal-link-button">
```

Comment:
- [ ] **[OPTIONAL] Your project looks awesome** :1st_place_medal: :1st_place_medal:, 
However, it would be better to make the following adjustments to your design  to replicate the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=9tObddwnmWVY0yAo-0) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏 
    - I think it would be better to add a gap between your buttons to match Figma design, you may use the  ```margin```  to add a space around the buttons.
    - It would be better also to add a space between ``` text and Icon``` and aligning them in center will make our project match that of Figma design,
        
I hope the following codes may help :joy: 
```
display: flex; 
gap: 24px; 
align-items: center;
```

**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/202841107-c135e30f-8acb-49cf-883d-41f1c448cd1f.png) |![image](https://user-images.githubusercontent.com/102757126/202841121-93cc7515-b248-42a4-b6a7-c613d972b5b5.png)

**KEEP IT UP! :+1:**